src/Worker/Targets/netstandard1.0/Microsoft.NET.Sdk.Worker.props

```diff
@@ -0,0 +1,36 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.Sdk.Web.ProjectSystem.props
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved.
+***********************************************************************************************
+-->
+
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <DebugSymbols Condition="'$(DebugSymbols)' == ''">true</DebugSymbols>
+    <DebugType Condition="'$(DebugType)' == ''">pdbonly</DebugType>
+    <GenerateDependencyFile Condition="'$(GenerateDependencyFile)' == ''">true</GenerateDependencyFile>
+    <IsPackable Condition="'$(IsPackable)' == ''">false</IsPackable>
```

Comment:
Do you want the worker project to be packable? For web, we didn't want to but I dont see a reason why it should be false for worker.

Reply:
Why aren't web projects packable? 

Workers are publishable applications just like web apps. I'm not sure why you'd pack them.

Reply:
This has context - https://github.com/aspnet/websdk/issues/150. If you are fine with worker working the same as websdk, then I am fine.